src/metrics/graphite_metric.ts

```diff
@@ -1,12 +1,43 @@
-import { AbsractMetric, Datasource, MetricId  } from "./metric";
+import { AbsractMetric, Datasource, MetricId  } from './metric';
+
+import * as moment from 'moment';
 
 export class GraphiteMetric extends AbsractMetric {
 
+    private static QUERY_TIME_REGEX = /\&from=[^\&]+\&until=[^\&]+\&/;
+    private static MAX_DATA_POINTS_REGEX = /\&maxDataPoints=[^\&]/;
```

Comment:
I guess, there should be `+` in the end? Or do you need just one symbol after `=`? `/\&maxDataPoints=[^\&]/` -> `/\&maxDataPoints=[^\&]+/`

Reply:
fixed, thanks